Tests/CustomDumpTests/DiffTests.swift

```diff
@@ -103,11 +103,11 @@ final class DiffTests: XCTestCase {
       diff(
         User(),
         User()
-      )?.replacingOccurrences(of: "0x[[:xdigit:]]+", with: "…", options: .regularExpression),
+      )?.replacingOccurrences(of: "0x[[:xdigit:]]+", with: "0x…", options: .regularExpression),
       """
         DiffTests.User(
-      -   _: ObjectIdentifier(…),
-      +   _: ObjectIdentifier(…),
```

Comment:
Noticed this `…` is something the test substitutes in, but `…` is used by the library for collapsing diffs, so I've augmented it a bit to avoid confusion.